datajoint/autopopulate.py

```diff
@@ -48,7 +48,7 @@ def parent_gen(self):
             try:
                 self._key_source = next(parents)
             except StopIteration:
-                raise DataJointError('A relation must have primary dependencies for auto-populate to work') from None
+                raise DataJointError('A relation must have primary dependencies for auto-populate to work')
```

Comment:
```suggestion
                raise DataJointError('A table must have primary dependencies for auto-populate to work')
```
This remained from a time when we used theory-based terminology.